### ISSUE TITLE
f1 score low!!!

### ISSUE THREAD

**Issue by michaelzhouy** [jfpuget/STFT_Transformer#3]
Hi, CPMP. I knew this project from the kaggle. I want to learn from your work, so I ran your code. But I got very low f1 score in the first 3 epochs. Is this normal?

loss: 0.2049, smth: 0.1818: 100%|██████████| 13726/13726 [35:29<00:00,  6.44it/s]
loss: 0.0635, smth: 0.1129: 100%|██████████| 858/858 [03:15<00:00,  4.39it/s]
Orig 0 Ep 0, lr: 0.0001000, train loss: 0.28392, val loss: 0.21424, f1: 0.0784 0.0950 0.0830 0.0805 0.0779
Thu May  5 23:40:27 2022 Epoch: 1
loss: 0.1854, smth: 0.1726: 100%|██████████| 13726/13726 [35:25<00:00,  6.46it/s]
loss: 0.0494, smth: 0.0906: 100%|██████████| 858/858 [03:10<00:00,  4.50it/s]
Orig 0 Ep 1, lr: 0.0000970, train loss: 0.17728, val loss: 0.17907, f1: 0.0948 0.1055 0.0959 0.0811 0.0748
Fri May  6 00:19:09 2022 Epoch: 2
loss: 0.1628, smth: 0.1752: 100%|██████████| 13726/13726 [35:25<00:00,  6.46it/s]
loss: 0.0728, smth: 0.0919: 100%|██████████| 858/858 [03:08<00:00,  4.55it/s]
Orig 0 Ep 2, lr: 0.0000883, train loss: 0.17329, val loss: 0.15498, f1: 0.0941 0.1128 0.1118 0.0872 0.0765
Fri May  6 00:57:49 2022 Epoch: 3
loss: 0.1113, smth: 0.1657: 100%|██████████| 13726/13726 [35:30<00:00,  6.44it/s]
loss: 0.0391, smth: 0.0707: 100%|██████████| 858/858 [03:34<00:00,  4.00it/s]
Orig 0 Ep 3, lr: 0.0000750, train loss: 0.16898, val loss: 0.14133, f1: 0.1139 0.1333 0.1339 0.1080 0.0915

**Comment by michaelzhouy** [comment]
bs is low
